projects/plugins/jetpack/modules/contact-form/grunion-contact-form.php

```diff
@@ -1318,18 +1318,18 @@ public function get_post_content_for_csv_export( $post_id ) {
 	 * @return mixed
 	 */
 	public function get_post_meta_for_csv_export( $post_id ) {
-		$md                  = get_post_meta( $post_id, '_feedback_extra_fields', true );
-		$md['feedback_date'] = get_the_date( DATE_RFC3339, $post_id );
-		$content_fields      = self::parse_fields_from_content( $post_id );
-		$md['feedback_ip']   = ( isset( $content_fields['_feedback_ip'] ) ) ? $content_fields['_feedback_ip'] : 0;
+		$md                                  = get_post_meta( $post_id, '_feedback_extra_fields', true );
+		$md[ __( 'Date', 'jetpack' ) ]       = get_the_date( DATE_RFC3339, $post_id );
+		$content_fields                      = self::parse_fields_from_content( $post_id );
+		$md[ __( 'IP Address', 'jetpack' ) ] = ( isset( $content_fields['_feedback_ip'] ) ) ? $content_fields['_feedback_ip'] : 0;
 
 		// add the email_marketing_consent to the post meta.
-		$md['email_marketing_consent'] = 0;
+		$md[ __( 'Consent', 'jetpack' ) ] = 0;
```

Comment:
What do you think about adding context for those 2 strings, to help translators?

```suggestion
		$md[ _x( 'Consent', 'noun', 'jetpack' ) ] = 0;
```